android/.idea/.name

```diff
@@ -0,0 +1 @@
+VCMI
```

Comment:
Do we need these idea files?

Reply:
@nullkiller maybe kill all .idea folder?

Reply:
this folder should not be versioned, but be wary to backup this folder when pulling changes after this PR gets merged, so your settings do not get eaten... same goes for clion c++ devs

Reply:
@dydzio0614 agree.

Reply:
It was present historically, so lets remove.

Reply:
ok, will remove. Anyone can still pull it from the old Android repo.